### ISSUE TITLE
Change Request: Add links to Track 1 resources in README

### ISSUE THREAD

**Issue by ramya-rao-a** [Azure/azure-sdk#433]
## The Basics

* Which design guideline is affected?
README template

* Which target languages are affected?
All

* Describe the change:
Add a section that has links to Track 1 resources in the README if we are using the same package for Track 2

## Reasoning
Our customers would land in the "samples" folder by various different means
- Links from npmjs, nuget etc
- Quick starts
- Blogs
- Bookmarks
- Organically landing on the monorepo and digging through the service they are interested in.

For cases where we are using the same package between Track 1 and Track 2, our Track 1 customers can land in the "samples" folder of the master branch which will have the samples for Track 2. Same goes for the README which also has code snippets for Track 2.

This issue is to add a section for a disclaimer of sorts in the README that says
- This is for version a.b.c. If you are using a lower version of this library, please see <link to readme from the last commit to Track 1> and <link to samples from the last commit to Track 1>

The samples folder should also have a README with
- general instructions/pre-requisistes for running samples
- This is for version a.b.c. If you are using a lower version of this library, please see <link to samples from the last commit to Track 1>

cc @bterlson

**Comment by ramya-rao-a** [comment]
Example:

![image](https://user-images.githubusercontent.com/16890566/59888600-61a66280-937d-11e9-90b7-9da10cff21d6.png)

**Comment by adrianhall** [comment]
What's the business reason for needing the additional links?  We WANT people to be using the newer versions, and the branches / package managers already allow you to go down-rev. I'm not seeing the reason for this overall.

**Comment by ramya-rao-a** [comment]
Existing users will not be using the newer versions until they explicitly opt-in to do the major version update/move to new library which involves non trivial amount of work around changing their code.

This means that there would be a considerable amount of time even after Track 2 is released that we will have users using the Track 1 code. There are many cases of such existing users of the sdk landing on our READMEs or samples. Examples:
- There is this feature I never knew existed, but I just read in a blog or a co-worker told me about it. Now, I want to use that feature in my project to make it better
- I want to understand a particular api/sample scenario from this sdk better, because turns out I made wrong assumptions and now there is a bug in my code!!
- There is a new person joining my team . I want to give links to the sdk, so that the new joinee can go over the README and samples to understand the sdk better in order to understand the team project better.
- So on and so forth

Moving to track 2 is not the answer for any of the above scenarios

Please note that this request to add the links is **only** for the case where we are re-using the package name between Track 1 and Track 2. None of the above scenarios hold true when we are shipping a brand new library with a new name.

**Comment by adrianhall** [comment]
Agreed in principal.  Can you please do a PR against the repo and I'll merge it.
